ghc-lib-gen/src/Ghclibgen.hs

```diff
@@ -95,7 +101,7 @@ ghcLibParserIncludeDirs ghcFlavor =
     ]
 
 -- C-preprocessor "include dirs" for 'ghc-lib'.
-ghcLibIncludeDirs :: GhcFlavor -> [FilePath]
+ghcLibIncludeDirs :: Bool -> GhcFlavor -> [FilePath]
 ghcLibIncludeDirs = ghcLibParserIncludeDirs
```

Comment:
If this defers to ghcLibParserIncludeDirs, then this Bool argument is also unused, I think

Reply:
it is true that `forDepends` is ignored. it's ok that it should remain.